packages/web-app/src/components/appli/Entry/Locations/index.jsx

```diff
@@ -64,9 +64,11 @@ const Locations = ({ entranceId, locations }) => {
             </List>
           ) : (
             <Alert
-              severity="info"
-              title={formatMessage({
-                id: 'There is currently no location for this entrance.'
+              severity={isSensitive ? 'warning' : 'info'}
+              content={formatMessage({
+                id: isSensitive
+                  ? 'This entrance has a restricted access, you can not see its locations.'
```

Comment:
Il faut mettre locations au pluriel ici ?

Reply:
This entrance has a restricted access, the exact location is unavailable / hidden.

Reply:
Ici il s'agit des objets "Locations" à droite, et non des coordonnées. 
On a toujours fait la distinction "Coordinates" (latitude / longitude) et "Locations" (description de l'accès à la cavité). D'où mon utilisation du terme locations au pluriel ici.